example/k3d/smoke/main.jsonnet

```
@@ -86,6 +89,29 @@ local smoke = {
       configMap.mixin.metadata.withNamespace("smoke")
   },
 
+  loki: loki.new(namespace = 'smoke'),
+
+  // https://grafana.com/docs/loki/latest/operations/loki-canary/
+  canary : canary{
```

You might want to install jsonnetfmt :)

```suggestion
  canary: canary{
```

---

Thanks for the suggestion. Ran `:JsonnetFmt` and it added some linting fixes from previous PRs.